### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by kmcnellis**
Add text for new users to prompt actions and avoid confusion
Standardize headers / sizes on dashboard tabs
Improve menu and sidebar options
Add section and navigation to submission cards
Add new (empty) instructor seed

### CODE REVIEW

File: client/app/student/dashboard/dashboard.controller.js [hunk 1]
```diff
@@ -10,8 +10,12 @@ angular.module('venueApp')
     Auth.getCurrentUser((user) => {
       $scope.user = user;
 
-      User.get({id: user._id, withCourses:true, withEvents: true, withEventSections: true}, (usr) => {
-        $scope.user = usr;
+      User.get({id: user._id, withCourses:true, withEvents: true, withEventSections: true}, (user) => {
+        $scope.user = user;
+        $scope.coursesLength = angular.equals(user.courses,{})?0:1
+
+        $scope.eventsLength = angular.equals(user.events, {})?0:1
```
Comment (seveibar) [inline client/app/student/dashboard/dashboard.controller.js]: so `$scope.coursesLength` doesn't represent the number of courses?

File: client/components/sectionform/sectionform.controller.js [hunk 2]
```diff
@@ -22,7 +22,12 @@ angular.module('venueApp')
         }
       });
       $scope.section = $scope.course.sections.splice(currentSection, 1)[0];
-      $scope.section.sectionNumbers = $scope.section.sectionNumbers.toString();
+      $scope.section.sectionNumbersText = $scope.section.sectionNumbers.toString();
+      $scope.$watch('section.sectionNumbers', function(newValue, oldValue) {
+        console.log("Loaded section Numbers")
```
Comment (seveibar) [inline client/components/sectionform/sectionform.controller.js]: `console.log`

**Comment by seveibar** [comment]
Awesome, we had a ton of dead ends- i am confused at that variable name though- quick clarification on that?

**Comment by jowlee** [comment]
selection_010

**Comment by seveibar** [comment]
Also Joey and I are kinda of the opinion the gray doesn't really work- but we'll figure that out later (probably part of a larger redesign) :penguin:

File: client/app/student/dashboard/dashboard.controller.js (hunk 1 above)
Comment (kmcnellis) [inline client/app/student/dashboard/dashboard.controller.js]: It check if there is any. It evolved from the previous method when we had arrays (unfortunately its not efficient to directly check the number of keys in objects), and I forgot to change the variable name. I can change it

**Comment by kmcnellis** [comment]
I'm also not a fan of grey events. Mostly because those are the events that the instructors and clicking to grade and students submitting to. I would like a way to show that events are future (you can't submit yet), current (you can submit), and past (you can submit but we should mention it is a late submission if its more than a day late, and instructors should grade). Maybe also have a official period of time that submissions are allowed and after that, they get graded?

**Comment by seveibar** [comment]
cool- i guess ill take a look at the travis ci bug before merging

**Pull request merged**